src/main/java/no/sikt/nva/model/record/Record.java

```diff
@@ -197,6 +209,10 @@ public String getRightsholder() {
         return rightsholder;
     }
 
+    public void setRightsholder(String rightsholder) {
+        this.rightsholder = rightsholder;
+    }
+
     public void setRightsHolder(String rightsholder) {
         this.rightsholder = rightsholder;
     }
```

Comment:
Ser jeg syner eller er det to settere for setRightsHolder?

Reply:
det er det ja, fjerner